### ISSUE TITLE
False detect on `import discord` even discord.py is installed

### ISSUE THREAD

**Issue by PerchunPak** [PyCQA/pylint#4985]
### Bug description

Title
P.S. If i run code with python, there are no errors
P.S.2 Using [this repository](https://github.com/PerchunPak/PingerBot/tree/db64725bc964213fe2eb0783f23f69ef3236c5b4) to find it

### Command used

```shell
pylint `ls -R|grep .py$|xargs` -E
```

### Pylint output

```shell
************* Module commands
commands.py:1:0: E0401: Unable to import 'discord.ext.commands' (import-error)
commands.py:2:0: E0401: Unable to import 'discord' (import-error)
```

### Expected behavior

It will check not with installed modules, but with available folders to import (as it do python i think)
Or add argument --import-error-ignore

### Pylint version

```shell
pylint 2.10.2
astroid 2.7.3
Python 3.9.7 (default, Sep  4 2021, 03:09:52)
[Clang 9.0.8 (https://android.googlesource.com/toolchain/llvm-project 98c855489
```

**Comment by DudeNr33** [comment]
I was not able to reproduce the problem.
Here is what I did. Please correct me if your workflow differs.

```shell
git clone https://github.com/PerchunPak/PingerBot.git
cd PingerBot
pip install -r requirements.txt
pip install -U pylint
pylint `ls -R|grep .py$|xargs` -E
```

This only produces:
```shell
************* Module main
main.py:11:0: E0237: Assigning to attribute 'members' not defined in class slots (assigning-non-slot)
```

Can you confirm that you run ``pylint`` in the same Python environment where ``discord`` is installed?

**Comment by PerchunPak** [comment]
Strange but theems like something fixed it, anyway you found new bug because attribute 'members' doesnt give any errors when run program
